### ISSUE TITLE
Rice-video bugs with Zelda

### ISSUE THREAD

**Issue by richard42** [mupen64plus/mupen64plus-user-issues#541]
Originally reported on Google Code with ID 542

```
System:
 - Ubuntu 12.04:
 - 32bit:
 - 1.99.5:
 - Video-rice:

'THE LEGEND OF ZELDA', CRC 8a805705510051f0-45
Somthing really wrong...
```

Reported by `daniel.plachotich@yandex.ru` on 2013-01-28 05:03:26

<hr>
- _Attachment: 2013-01-28-065636_802x627_scrot.png<br>![2013-01-28-065636_802x627_scrot.png](https://storage.googleapis.com/google-code-attachments/mupen64plus/issue-542/comment-0/2013-01-28-065636_802x627_scrot.png)_

**Comment by richard42** [comment]
```
This thing happens only on 2d backgrounds.
```

Reported by `daniel.plachotich@yandex.ru` on 2013-01-28 05:05:04

**Comment by richard42** [comment]
```
Same as issue 494:
http://code.google.com/p/mupen64plus/issues/detail?id=494

Problem is fixed in latest HG versions.
```

Reported by `bobby.smiles32` on 2013-01-28 18:34:16

**Comment by richard42** [comment]
Reported by `sven@narfation.org` on 2013-01-28 21:51:35
- **Status changed**: `Fixed`

**Comment by richard42** [comment]
Reported by `sven@narfation.org` on 2013-01-28 21:51:54
- **Status changed**: `Duplicate`
- **Merged into**: #494
